docassemble/juvenilesealing/data/questions/sealing-body.yml

```diff
@@ -1,4 +1,40 @@
 ---
+metadata:
+  description: |
+    This interview helps you figure out if you can seal your juvenile record.
+    If you can, it will help you fill out the form to do that.
+  can_I_use_this_form: |
+    You can use this form if you have a juvenile record in Massachusetts.
+  before_you_start: |
+    Before you start, you should know that:
+    
+    - **Sealing** your record is not the same as **expunging** it. When you seal your record, it is hidden from most people, but not everyone.
+    - **Sealing** your record means that most employers will not see it. Only employers who do federal background checks will see it. Also, people like police and judges will be able to see your criminal record.
+    - **Sealing** your record can help you get a job, housing, and other things.
+    - **Sealing** your record can take a few months. It can take up to 3 months to get a letter from the Commissioner of Probation and sometimes takes a few weeks more for the courts to finish the paperwork.
+    - **Sealing** your record is not the same as sealing your CORI. This form is only for sealing your juvenile record.
+
+  maturity: production
+  estimated_completion_minutes: 40
+  estimated_completion_delta: 40
+  languages: 
+    - en
+  help_page_url: https://www.gbls.org/self-help/cori
+  help_page_title: CORI and Juvenile Record Sealing
```

Comment:
```suggestion
  help_page_title: Juvenile Record Sealing
```

Reply:
Fwiw, this was the exact title of the page we link to here. But I'm fine with this change